README.md

```diff
@@ -21,8 +21,8 @@ It is intended for use with editor/IDE integration.
 
 ## Supported linters
 
-- [go vet](http://godoc.org/golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
-- [gotype](golang.org/x/tools/cmd/gotype) - Syntactic and semantic analysis similar to the Go compiler.
+- [go vet](https:///golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
```

Comment:
Yeah I'm not sure what the canonical location for it is (ie. the github repo says "mirror"), but I guess http://golang.org/cmd/vet/ would be preferable?

PS. Thanks for catching these documentation bugs, appreciate it.